integreat_cms/cms/views/language_tree/language_tree_node_form_view.py

```diff
@@ -42,3 +43,33 @@ def get_form_kwargs(self):
         kwargs = super().get_form_kwargs()
         kwargs["region"] = self.request.region
         return kwargs
+
+
+class LanguageTreeNodeUpdateView(CustomUpdateView):
+    """
+    Class that handles activating/deactivating of a language tree node
+    """
+
+    def post(self, request, *args, **kwargs):
+        language_tree_node = self.get_object()
+        language_tree_form = LanguageTreeNodeForm(
+            data=request.POST, instance=language_tree_node
+        )
+
+        if "active" in language_tree_form.changed_data:
+            region = language_tree_node.region
+            language = language_tree_node.language
+
+            models = [PageTranslation, EventTranslation, POITranslation]
+            for model in models:
+                filters = {
+                    f"{model.foreign_field()}__region": region,
+                    "language": language,
+                }
+                for translation in model.objects.filter(**filters):
+                    if request.POST.get("active") == "on":
```

Comment:
Similar to above: I don't think it's a good idea to rely on the sent post data, instead I would wait for Django to validate and save all the data, and then access e.g. `self.get_object().first().active` to determine whether the selected nodes have been enabled or disabled.